manifests/cinder-csi-plugin/cinder-csi-controllerplugin.yaml

```
@@ -73,6 +74,7 @@ spec:
           image: quay.io/k8scsi/csi-resizer:v0.4.0
           args:
             - "--csi-address=$(ADDRESS)"
+            - "--timeout=3m"
```

For `quay.io/k8scsi/csi-resizer:v0.4.0`, the flag is still `--csiTimeout` (it is changed to `--timeout` since external-resizer@1.0.0).

```suggestion
            - "--csiTimeout=3m"
```

```
$ docker run quay.io/k8scsi/csi-resizer:v0.4.0 --help

  -csiTimeout duration
    	Timeout for waiting for CSI driver socket. (default 15s)
```

---

@ialidzhikov thanks for a hint

---

@ramineni we need to keep this in mind, when update the csi-resizer tag next time.